R/FilterStateDate.R

```diff
@@ -1,17 +1,25 @@
+# DateFilterState ------
+
 #' @name DateFilterState
-#' @title `FilterState` object for Date variable
-#' @description Manages choosing a range of Dates
 #' @docType class
-#' @keywords internal
 #'
+#' @title `FilterState` object for Date variable
+#'
+#' @description Manages choosing a range of Dates.
```

Comment:
@kartikeyakirar should `Date` in title and in description be included in backticks as this is a class in R?
Like you type `\code{character}` I think we should also type `\code{Date}` but use backticks instead of `\code{}` (I'm using \code{} only because backticks are reserved on GitHub markdown.

Reply:
[14f1415](https://github.com/insightsengineering/teal.slice/pull/506/commits/14f14151f3e5160b99f83942eabeac347de53591)